source/js/main.js

```diff
@@ -203,6 +203,16 @@ let main = {
         Analytics.sendGAEvent(`navigation`, `page footer cta`, label);
       });
     }
+
+    if (document.querySelector(`#view-participate .card-cta .btn[href*="donate.mozilla.org"]`)) {
+      document.querySelector(`#view-participate .card-cta .btn[href*="donate.mozilla.org"]`).addEventListener(`click`, () => {
```

Comment:
Can this be saved as a variable so we don't have to fetch it twice? Not necessarily from a performance standpoint, but from a readability one.

Reply:
Yea I thought about doing this but decided to go with what we already have in the rest of the code. (No variable is declared for any DOM nodes.) Maybe we should file a followup ticket to tidy things up all at once?